### PR TITLE
doc: verification tests

### PR DESCRIPTION
**Pull request by dgarus**
part of [445](https://github.com/artipie/asto/issues/445)

Added a verification tests description to the storage section.

### CODE REVIEW

File: .wiki/Configuration-Storage.md [hunk 1]
```diff
@@ -111,6 +111,20 @@ Initiated storage factory [type={your-storage-type}, class={your-storage-factory
 You can study [a storage implementation based on Redis java client Redisson](https://github.com/artipie/asto/tree/master/asto-redis/src/main/java/com/artipie/asto/redis)
 as a good example.
 
+# Storage Verification tests
+
+The `asto-core` module contains verification tests that designed to confirm correctness of a storage implementation.
```
Comment (olenagerasimova) [inline .wiki/Configuration-Storage.md:116]: that -> which are

File: .wiki/Configuration-Storage.md [hunk 2]
```diff
@@ -111,6 +111,20 @@ Initiated storage factory [type={your-storage-type}, class={your-storage-factory
...
+A storage verification test class have to extend the `StorageWhiteboxVerification` and provide the `newStorage` method,
```
Comment (olenagerasimova) [inline .wiki/Configuration-Storage.md:117]: have -> has
provide -> implement

File: .wiki/Configuration-Storage.md [hunk 3]
```diff
@@ -111,6 +111,20 @@ Initiated storage factory [type={your-storage-type}, class={your-storage-factory
...
+which returns a new instance of storage to check.
+
+> Note: To avoid affect one test to another, test environment should be returned to initial state before each test.
```
Comment (olenagerasimova) [inline .wiki/Configuration-Storage.md:120]: affect one test to another -> possible mutual affects between tests methods

File: .wiki/Configuration-Storage.md [hunk 4]
```diff
@@ -111,6 +111,20 @@ Initiated storage factory [type={your-storage-type}, class={your-storage-factory
...
+
+> Note: If storage can't be used with [SubStorage](https://www.javadoc.io/static/com.artipie/asto-core/v1.13.0/com/artipie/asto/SubStorage.html),
+> in that case you should override `newBaseForSubStorage` or(and)
```
Comment (olenagerasimova) [inline .wiki/Configuration-Storage.md:123]: "in that case" is not needed here

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@dgarus thanks, check my comments please

**Comment by dgarus** [comment]
@olenagerasimova thanks for the reviw!
I fixed comments. Also, I removed `Rultor.com` label from readme file.

**Review by olenagerasimova — APPROVED** [review]
@dgarus thanks

**Pull request merged**